<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMMGeneratorBoundaryConditionsEditorWidget</class>
 <widget class="QFrame" name="CMMGeneratorBoundaryConditionsEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>173</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMMGeneratorBoundaryConditionsEditorWidget</string>
  </property>
  <layout class="QFormLayout" name="mainLayout">
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblMinMatrixSize">
     <property name="text">
      <string>Min matrix size:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sbMinMatrixSize">
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblMaxMatrixSize">
     <property name="text">
      <string>Max matrix size:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbMaxMatrixSize">
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblMatrixSizeIncrement">
     <property name="text">
      <string>Matrix size increment:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbMatrixSizeIncrement">
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblBlockSize">
     <property name="text">
      <string>Block size:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbBlockSize">
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>1048576</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
